lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -188,6 +193,19 @@ def find_dir_containing_file(directory, target)
         end
         result
       end
+
+      private
+
+      def check_version(name)
+        current_version = package_version(name)
+        if current_version
+          yield current_version if block_given?
+        end
+      end
+
+      def version_cmp(version, other_version)
```

Comment:
For readability purposes, actually I think this method is redundant, and just keeping `>=` in the `check_{min,max}_version` methods would be easier to understand.